css-canvas-medical/styles.css

```diff
@@ -0,0 +1,219 @@
+.container {
+  max-width: 800px;
```

Comment:
I would widen the container to about 1200px, that should be solid for a desktop size and will give property horizontal space.